pom.xml

```diff
@@ -101,12 +99,6 @@
       </dependency>
 
       <dependency>
```

Comment:
see the dependency just above - hawkular-accounts-websocket-api - this was added recently for use by feed-comm. Now that that has been split out, I believe this can be removed (and should be in the new command-gateway pom)